scripts/affected_targets

```diff
@@ -14,43 +14,78 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Finds all targets of a given kind affected by changes since origin/develop.
+# Finds all targets of a given kind affected by changes in a given commit range.
+
+SCRIPT_NAME=$(basename "$0")
 
 # type should be a bazel `kind` type. E.g. test or rule.
 # https://docs.bazel.build/versions/master/query.html#kind
 type="$1"
```

Comment:
Tip: Maybe in a follow-up, standardize on the name `bazel_kind`? That makes more sense now that I see the comment/docs. :)

Reply:
Done.